content/en/blog/2021/revision-tags/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Istio control plane canary with revisions and tags"
+description: Learn how to take more control over your Istio control-plane upgrades.
+publishdate: 2021-05-20
+attribution: "Christian Posta (Solo.io), Lin Sun (Solo.io), Sam Naser (Google)"
+keywords: [upgrades,revisions,operations,canary]
+---
+
+The Istio community [releases new versions of Istio](/about/supported-releases/) every quarter and sometimes between the normal release cadence for bug fixes [or CVE patches](/about/security-vulnerabilities/). The operator of an Istio service mesh needs to upgrade the control plane and data plane components during the life of the service mesh. You must take care when upgrading a service mesh as a mistake could take down the entire mesh and affect your business traffic. Istio has a couple mechanisms to make it safe to perform upgrades in a controlled manner, and in Istio 1.10 we further improve this operational experience.
+
+## Background
+
+In Istio 1.6, [we added basic support for upgrading the service mesh](/news/releases/1.6.x/announcing-1.6/change-notes/) following a [canary pattern using revisions](/blog/2020/multiple-control-planes/). Using this approach, you can run multiple control planes side-by-side without impacting an existing deployment and slowly migrate workloads from an old control plane to the new control plane.
+
+To support this revision-based upgrade, Istio introduced the `istio.io/rev` label for namespaces which indicates to which control plane the workload data-plane proxies in that namespace should connect. For example, a label of `istio.io/rev=1-9-3` indicates the control plane revision `1-9-3` should inject the data plane for workloads in that namespace.
+
+If you wanted to upgrade the data-plane proxies for a particular namespace, you would update the `istio.io/rev` label to point to a new version such as `istio.io/rev=1-10-0`. Across many namespaces, updating labels one at a time could be tedious and error prone. Let's see how Istio can improve your experience doing upgrades.
```

Comment:
```suggestion
To support this revision-based upgrade, Istio introduced a `istio.io/rev` label for namespaces. This indicates which control plane the sidecar proxies in that namespace should connect to. For example, a label of `istio.io/rev=1-9-3` indicates the control plane revision `1-9-3` should inject the data plane for workloads in that namespace.

If you wanted to upgrade the data-plane proxies for a particular namespace, you would update the `istio.io/rev` label to point to a new version, such as `istio.io/rev=1-10-0`. Manually changing (or even trying to orchestrate) changes of labels across a large number of namespaces can be error-prone and lead to unintended downtime.
```

I'm not sure about the first sentence.  It talks first about _what control plane the proxy should connect to_, but then switches to _what version should be injected_.  These two should probably be the same, but it's not clear in this text that they are.

Also moved a sentence up from below.